81-IDevID_and_IAK_support.md

```diff
@@ -173,8 +173,16 @@ bogged down.
 -->
 
 #### Story 1
+<!-- TODO: add IDevID scenario with Keylime using it to make sure the device is authentic. -->
+In order to securely identify that a device is genuine from a given OEM the IDevID can play an important role.
+Within the registration process, along with information like the EK and EAK, the IDevID key and certificates will be provided.
+At the registrar service the OEM Root CA will be configured and the IDevID trust chain will be checked, ensuring the certificates are signed by the OEM CAs. 
+In addition to the certificates the signed IDevID certificate Subject will contain the device's SN(Serial Number) and the registrar service will check it against the information obtained from the device.
 
 #### Story 2
+<!-- TODO: add LDevID scenario with the user creating its own identity. -->
+With the IAK provided by the OEM, the client will issue it's own LDevID signed by it's own CAs. Instead of a long lived certificate as the IDevID from the OEM, the client can renew this certificate on a more frequent basis and securely verify its binding to the IAK and the devices TPM. 
```

Comment:
"the client will issue it's own LDevID signed by it's own CAs". This makes it sounds like the client is forced to do it.

Consider "can" instead of "will":

"the client can issue it's own LDevID signed by it's own CAs"

Reply:
ok, replaced with "can" both at the LDevID and LAK cases.